contribs/gnofaucet/.env.example

```diff
@@ -0,0 +1,7 @@
+# required
+# These credentials are obtained when registering an application on GitHub at:
+# https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/authenticating-to-the-rest-api-with-an-oauth-app#registering-your-app
+GH_CLIENT_SECRET=
+REDIS_ADDR=
```

Comment:
Document redis usage

Reply:
did you meant something like this 14157777b1f3e104cb8eae8263241a195454a72a ?